Dockerfile.production

```diff
@@ -37,9 +38,24 @@ RUN rm -rf node_modules spec
 
 RUN bundle exec bootsnap precompile --gemfile app/ lib/
 
```

Comment:
Add a suitable comment for this build stage

Reply:
I have added a comment can you please review ?